net-print/cnijfilter-ip7200/cnijfilter-ip7200-3.80.ebuild

```diff
@@ -0,0 +1,113 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools
+
+MY_P="cnijfilter-source-${PV}-1"
+
+DESCRIPTION="CUPS drivers for Canon Pixma inkjet printers (printer specific part)"
+HOMEPAGE="https://wiki.gentoo.org/wiki/Canon_Pixma_Printer"
+SRC_URI="http://gdlp01.c-wss.com/gds/3/0100004693/01/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
Are you convinced this is GPL-2? Because AFAICS there's an EULA in `LICENSE*` which seems to cover the prebuilt files.

Reply:
No, you are right, there is a Canon prologue before the GPL v2 not only in the license files, but also displayed on the download web pages. Should i submit that prologue as a new license in Gentoo ?